inc/class-formatter.php

```diff
@@ -70,6 +88,25 @@ private function show_table( $items, $fields, $include_total ) {
 		if ( ! is_null( $this->total_cell_index ) ) {
 			$totals[ $this->total_cell_index ] = 'total (' . count( $items ) . ')';
 		}
+
+		if ( $orderby ) {
+			usort( $items, function( $a, $b ) use ( $order, $orderby ) {
+				if ( 'ASC' === $order ) {
```

Comment:
Instead of having two separate code paths for the two different sort orders, you could also swap them as needed, and then have 1 unified code path after that.

Something like this:
```php
usort( $items, function( $a, $b ) use ( $order, $orderby ) {
   list( $first, $second ) = 'ASC' === $order ? array( $a, $b ) : array( $b, $a );

   if ( is_numeric( $first->$orderby ) && is_numeric( $second->$orderby ) ) {
      return $this->compare_float( $first->$orderby, $second->$orderby );
   }

   return strcmp( $first->$orderby, $second->$orderby );
});
```